### ISSUE TITLE
Dodanie Fedrowania do Google Analytics

### ISSUE THREAD

**Issue by Roksana90** [watchdogpolska/feder#1100]
Hej,
możemy dodać Fedrowanie do śledzenia w Google Analytics?

**Comment by ad-m** [comment]
Tak.

Jakim kodem?

**Comment by Roksana90** [comment]
Chyba takim G-547HV69QMQ

**Comment by mrbojko** [comment]
@ad-m oto kod:
`
<script async src="https://www.googletagmanager.com/gtag/js?id=G-547HV69QMQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-547HV69QMQ');
</script>`

**Comment by ad-m** [comment]
Zagadnienie zablokowane przez konieczność rozstrzygnięcia kwestii prawnych.

**Comment by mrbojko** [comment]
@ad-m co masz na myśli? Podłączanie w ogóle GA, czy coś związanego z kodem Fedrowania? Czy rozważamy użycie innego narzędzia analitycznego?

**Comment by dzejkobi** [comment]
@ad-m Czy chodzi o kwestię umieszczenia id Google w kodzie? Jeśli tak, to można skonfigurować go jako zmienną środowiskową.

**Comment by ad-m** [comment]
W kontekście omawianych dziś na spotkaniu zespołu wyzwań co do ochrony danych, szerzej przyglądamy się Fedrowaniu w zakresie danych osobowych i polityki cookies wraz z obsługą prawną Stowarzyszenia. Jak zaprojektujemy nowe podejście, które uwzględnia wymagania prawne to podesłany kod zostanie wprowadzony. Dotąd Fedrowanie używało cookies wyłącznie w celu zapewnienia funkcjonowania, a Google Analytics to zmienia.

**Comment by ad-m** [comment]
> @ad-m Czy chodzi o kwestię umieszczenia id Google w kodzie? Jeśli tak, to można skonfigurować go jako zmienną środowiskową.

Nie. Naturalnym jest użycie zmiennej środowiskowej, zwłaszcza że mamy dwie instancje.

**Comment by olekstomek** [comment]
A może użycie [Matomo ](https://github.com/matomo-org/matomo) (dawniej PIWIK)?

>  Matomo is the leading open alternative to Google Analytics that gives you full control over your data.

**Comment by ad-m** [comment]
Znam Matomo, ale jego wprowadzenie wymaga szerszego rozważenia w kontekście całości organizacji. Dotąd wykorzystywany był Google Analytics, mamy w tym zakresie kompetencje i procesy. Zmiana na Matomo wymaga jego wdrożenia – oceny potrzeb organizacji, zapewnienia kompetencji, instalacji oprogramowania, zaadaptowania pozostałych stron Stowarzyszenia.

**Comment by AgnieszkaZdanowicz** [comment]
@Roksana90 na czym stanęło z tym GA? Bo dyskusję pamiętam, a wniosków już nie.

**Comment by mrbojko** [comment]
W zasadzie to samo się rozwiąże. Google wysyła takie informacje:

<html><body>


Niedługo skonfigurujemy dla Ciebie Google Analytics 4
--
Dzień dobry,  1 lipca 2023 r. standardowe usługi Universal Analytics przestaną  przetwarzać dane. Aby zmaksymalizować ilość danych historycznych i mieć  pewność, że Google Analytics 4 (GA4) będzie odpowiadać potrzebom Twojej  firmy, przejdź już teraz na Google Analytics 4. Dzięki temu będziesz mieć możliwość dostosowania konfiguracji, w tym wdrożenia najnowszego tagu witryny.
W  marcu skonfigurujemy usługę GA4 z podstawowymi ustawieniami na kontach  klientów, którzy jeszcze nie przeszli na nową wersję. Wykorzystamy do  tego podstawowe ustawienia z ich wcześniejszej usługi Universal  Analytics, w tym niektóre zdarzenia konwersji i połączenia z kontem  Google Ads. Użyjemy także bieżących tagów witryny.  Dla klientów używających Universal Analytics oraz tagów witryny gtag.js  lub analytics.js (w tym ustawionych przez system zarządzania tagami taki  jak Menedżer tagów Google) utworzymy połączony tag witryny, który w  miarę możliwości będzie wykorzystywać wcześniejsze tagi do przesyłania  danych o ruchu do nowej usługi GA4. Jeśli nie chcesz, żebyśmy utworzyli  dla Ciebie usługę Google Analytics 4 i skonfigurowali ją na podstawie  bieżących tagów i ustawień usługi Universal Analytics, możesz z tego zrezygnować.


</body>
</html>Niedługo skonfigurujemy dla Ciebie Google Analytics 4

Dzień dobry,

1 lipca 2023 r. standardowe usługi Universal Analytics przestaną przetwarzać dane. Aby zmaksymalizować ilość danych historycznych i mieć pewność, że Google Analytics 4 (GA4) będzie odpowiadać potrzebom Twojej firmy. Dzięki temu będziesz mieć możliwość dostosowania konfiguracji, w tym wdrożenia najnowszego tagu witryny.

W marcu skonfigurujemy usługę GA4 z podstawowymi ustawieniami na kontach klientów, którzy jeszcze nie przeszli na nową wersję. Wykorzystamy do tego podstawowe ustawienia z ich wcześniejszej usługi Universal Analytics, w tym niektóre zdarzenia konwersji i połączenia z kontem Google Ads. Użyjemy także bieżących tagów witryny.

Dla klientów używających Universal Analytics oraz tagów witryny gtag.js lub analytics.js (w tym ustawionych przez system zarządzania tagami taki jak Menedżer tagów Google) utworzymy połączony tag witryny, który w miarę możliwości będzie wykorzystywać wcześniejsze tagi do przesyłania danych o ruchu do nowej usługi GA4. Jeśli nie chcesz, żebyśmy utworzyli dla Ciebie usługę Google Analytics 4 i skonfigurowali ją na podstawie bieżących tagów i ustawień usługi Universal Analytics,
